templates/control

```diff
@@ -3,6 +3,6 @@ Version: {{ node_deb_package_version }}
 Section: base
 Priority: optional
 Architecture: all
-Depends: nodejs
+Depends: nodejs , {{ node_deb_package_dependencies }}
```

Comment:
This line breaks installation if the `node_deb_package_dependencies` is undefined because `jq` reads the path as `null` and you get the following in the control file:

```
Depends: nodejs, null
```

Output from install:
```
dpkg: dependency problems prevent configuration of upstart-project:
 upstart-project depends on null; however:
  Package null is not installed.
dpkg: error processing package upstart-project (--install):
 dependency problems - leaving unconfigured
```

What this should do is check for the value in the `package.json` and if it returns the string `null`, set it to the default of `nodejs`, otherwise whatever the returned string is. I think also it would be a good idea to do a complete override on the `Depends` section. If a string is provided, nix the `nodejs` that's already there and let the user supply the full list they want. Another option would be to rename this `node_deb_package_additional_dependencies` so it's clear that it's not an override.

I'm open to suggestions on this. What do you think?